crates/rpc/proto/zed.proto

```diff
@@ -48,145 +48,147 @@ message Envelope {
+        OnTypeFormattingResponse on_type_formatting_response = 108;
+
+        UpdateWorktreeSettings update_worktree_settings = 109;
+
+        InlayHints inlay_hints = 110;
+        InlayHintsResponse inlay_hints_response = 111;
+        ResolveInlayHint resolve_inlay_hint = 112;
+        ResolveInlayHintResponse resolve_inlay_hint_response = 113;
+        RefreshInlayHints refresh_inlay_hints = 114;
+
+        CreateChannel create_channel = 115;
+        CreateChannelResponse create_channel_response = 116;
+        InviteChannelMember invite_channel_member = 117;
+        RemoveChannelMember remove_channel_member = 118;
+        RespondToChannelInvite respond_to_channel_invite = 119;
+        UpdateChannels update_channels = 120;
+        JoinChannel join_channel = 121;
+        DeleteChannel delete_channel = 122;
+        GetChannelMembers get_channel_members = 123;
+        GetChannelMembersResponse get_channel_members_response = 124;
+        SetChannelMemberRole set_channel_member_role = 125;
+        RenameChannel rename_channel = 126;
+        RenameChannelResponse rename_channel_response = 127;
+
+        JoinChannelBuffer join_channel_buffer = 128;
+        JoinChannelBufferResponse join_channel_buffer_response = 129;
+        UpdateChannelBuffer update_channel_buffer = 130;
+        LeaveChannelBuffer leave_channel_buffer = 131;
+        UpdateChannelBufferCollaborators update_channel_buffer_collaborators = 132;
+        RejoinChannelBuffers rejoin_channel_buffers = 133;
+        RejoinChannelBuffersResponse rejoin_channel_buffers_response = 134;
+        AckBufferOperation ack_buffer_operation = 135;
+
+        JoinChannelChat join_channel_chat = 136;
+        JoinChannelChatResponse join_channel_chat_response = 137;
+        LeaveChannelChat leave_channel_chat = 138;
+        SendChannelMessage send_channel_message = 139;
+        SendChannelMessageResponse send_channel_message_response = 140;
+        ChannelMessageSent channel_message_sent = 141;
+        GetChannelMessages get_channel_messages = 142;
+        GetChannelMessagesResponse get_channel_messages_response = 143;
+        RemoveChannelMessage remove_channel_message = 144;
+        AckChannelMessage ack_channel_message = 145;
+        GetChannelMessagesById get_channel_messages_by_id = 146;
+
+        MoveChannel move_channel = 149;
+        SetChannelVisibility set_channel_visibility = 150;
+
+        AddNotification add_notification = 151;
+        GetNotifications get_notifications = 152;
+        GetNotificationsResponse get_notifications_response = 153;
+        DeleteNotification delete_notification = 154;
+        MarkNotificationRead mark_notification_read = 155;
+        LspExtExpandMacro lsp_ext_expand_macro = 156;
+        LspExtExpandMacroResponse lsp_ext_expand_macro_response = 157;
+        SetRoomParticipantRole set_room_participant_role = 158;
+
+        UpdateUserChannels update_user_channels = 159; // current max
     }
 
-    reserved 158 to 161;
```

Comment:
I couldn't find anything about why we are reserving these specific Protobuf messages, I'm assuming they are just supposed to be filling the gap between `146 - 149` but wasn't sure. So just went with bumping this

Reply:
Sweet, can we leave 158 to 161 as reserved though? I think there are still versions of Zed in the wild that can send/receive these messages, so 162 and 163 should be a safer bet. Also, that current max comment is a bit redundant, as the list is already sorted by id. Other than that this looks great!

Reply:
Done it :)